rmw_iceoryx_cpp/src/internal/iceoryx_name_conversion.cpp

```diff
@@ -101,12 +77,45 @@ get_name_n_type_from_iceoryx_service_description(
     return std::make_tuple(instance, service);
   } else {
     // ARA Naming
-    std::string delimiter_msg = "_ara_msgs/msg/";
+    std::string service_lowercase = service;
```

Comment:
Can't comment on the lines above, but can the two string in line 72, 73 be deleted? They don't look like being used at all in this function.